projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -1582,12 +1582,6 @@ Optional<JobExecution> getJobExecutionStatus(V1Job job, JobStatusCondition jobSt
     jobExecutionStatusBuilder.succeeded(
         Optional.ofNullable(jobStatusCondition).map(JobStatusCondition::isSuccess).orElse(null));
 
-    // omits events that come after the Data Job completion
```

Comment:
That's because I made UserError a final status, which means that you cannot overwrite this status. The events which are not completed and come after the Data Job completion are marked as PlatformError so they cannot overwrite the UserError. The end status of the DataJobOOMIT is UserError so that's why it is still passing.

Reply:
why is platform_error not a final status?

Reply:
are you aware that DataJobOOMIT is commented out in main? 
I disabled it because it is very flakey. 
Would you be able to commented it back in in this PR. 
Because this is very likely to break it.

Reply:
Good catch, thanks. I will enable it.

Reply:
> why is platform_error not a final status?
Because in case of Platform Error the job execution is being restarted and in some of the subsequent executions it can Succeed, UserError, etc.